### DISCUSSION TITLE
Hard-coded PS coherence in integratePS.py

### DISCUSSION THREAD

**Discussion by stoormgeo** [isce-framework/fringe#77] [Q&A]
The final coherence (tcorr_ps_ds.bin) of the PS targets is constantly been 0.95. Then in Mintpy I can exclude PS targets by mask but trying to figure out why PS target integration is needed in the current implementation.
Thx.

**Comment by scottstanie** [comment]
Hi,
I believe the general idea of the workflow is
- the PS pixels are the most reliable pixels. Their phase shouldn't be altered by multilooking
- After all the other noisier pixels have been processed by the DS processing (meaning the SHP test using `nmap`, and the multilooking), we insert back the phase from the most reliable pixels where they existed.


I believe the 0.95 was picked as "some very high temporal coherence" in case you wanted to mask out low coherence pixels later. There's no significance to 0.95 exactly, AFAIK.
